src/mscorlib/ref/Compat/mscorlib.cs

```diff
@@ -6694,6 +6872,7 @@ public abstract partial class Calendar : System.ICloneable
         public abstract System.DateTime ToDateTime(int year, int month, int day, int hour, int minute, int second, int millisecond, int era);
         public virtual int ToFourDigitYear(int year) { return default(int); }
     }
+#if FEATURE_COREFX_GLOBALIZATION
```

Comment:
Yes except we know that some types won't be available in Unix like WinRT types for example. These #ifdefs I got while trying to mimic what coreclr currently does with it's reference assembly.